### PR TITLE
feat: Add Vue Table demo

### PR DESCRIPTION
**Pull request by nikhil-varma**
- This PR adds a new demo with Vue Table

Loom: https://www.loom.com/share/5fbba9122d6744c69d417f156523935d
image
CodeSandbox: https://codesandbox.io/s/tender-sunset-jlwf9

### CODE REVIEW

File: packages/vue-searchbox/examples/with-vue-table/src/App.vue [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+    return {
+      columns: [
+        "title",
+        "authors",
+        "average_rating",
+        "language_code",
+        "isbn",
+        "ratings_count"
+      ],
+      options: {
```
Comment (bietkul) [inline packages/vue-searchbox/examples/with-vue-table/src/App.vue]: @nikhil-varma Are you using it?

File: packages/vue-searchbox/examples/with-vue-table/src/App.vue [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+        // see the options API
+      }
+    };
+  },
+  methods: {
+    isChecked(value, key) {
```
Comment (bietkul) [inline packages/vue-searchbox/examples/with-vue-table/src/App.vue]: @nikhil-varma Please copy/paste carefully, remove the unused code.

File: packages/vue-searchbox/examples/with-vue-table/src/App.vue [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+      return value ? value.includes(key) : false;
+    },
+    handleChange(e, value, setValue) {
```
Comment (bietkul) [inline packages/vue-searchbox/examples/with-vue-table/src/App.vue]: same here, you're not using it

**Review by bietkul — CHANGES_REQUESTED** [review]

File: packages/vue-searchbox/examples/with-vue-table/src/App.vue (hunk 1 above)
Comment (nikhil-varma) [reply packages/vue-searchbox/examples/with-vue-table/src/App.vue:134]: It's the same code which you had in the demo.. So I just focused on the table rendering...

File: packages/vue-searchbox/examples/with-vue-table/src/App.vue (hunk 2 above)
Comment (nikhil-varma) [reply packages/vue-searchbox/examples/with-vue-table/src/App.vue:140]: Same here

File: packages/vue-searchbox/examples/with-vue-table/src/App.vue (hunk 3 above)
Comment (nikhil-varma) [reply packages/vue-searchbox/examples/with-vue-table/src/App.vue:143]: Same here

**Review by bietkul — APPROVED** [review]

**Pull request merged**